pingo/compat.py

```diff
@@ -0,0 +1,28 @@
+import sys
```

Comment:
Why this name `compat.py`? compat = compatibility?

I don't know if you followed the early discussion. There was a consensus on supporting Python3 only if Python2 compatibility is assured. Read more on Py3k support #45